src/main/java/core/basesyntax/service/activitieswithfruitservice/impl/PurchaseFruitActivitiesServiceImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.service.activitieswithfruitservice.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.activitieswithfruitservice.ActivitiesWithFruitService;
+
+public class PurchaseFruitActivitiesServiceImpl implements ActivitiesWithFruitService {
+    @Override
+    public void performTransaction(FruitTransaction fruitTransaction) {
```

Comment:
![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/bfb1f247-6790-474e-8c66-02feab964ad5)

Reply:
not fixed. you can't just ignore Mentor's suggestions. if you have any questions or need some help - they should be asked in the platform's chats